src/adapter/src/coord/message_handler.rs

```diff
@@ -146,13 +148,9 @@ impl<S: Append + 'static> Coordinator<S> {
             .storage_usage_collection_interval
             .saturating_sub(Duration::from_millis(time_since_previous_collection));
         let internal_cmd_tx = self.internal_cmd_tx.clone();
-        let now_fn = self.now_fn();
         task::spawn(|| "storage_usage_collection", async move {
             tokio::time::sleep(next_collection_interval).await;
-            if internal_cmd_tx
-                .send(Message::StorageUsageFetch(now_fn()))
-                .is_err()
-            {
+            if internal_cmd_tx.send(Message::StorageUsageFetch).is_err() {
```

Comment:
@mjibson I know this is already merged, but I don't think we want to continue using `self.now()` on line 144 to schedule the next collection. Instead we should probably use `self.peek_local_write_ts()`